addons/sale_expense/views/product_view.xml

```diff
@@ -21,6 +21,17 @@
         </field>
     </record>
 
+    <record id="sale_expense_product_template_form_view_expense_policy" model="ir.ui.view">
+        <field name="name">sale.expense.product.template.form.view.expense.policy</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="sale.product_template_form_view_invoice_policy"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='expense_policy']" position="attributes">
+                <attribute name="attrs">{'invisible': ['|', ('visible_expense_policy', '=', False), ('can_be_expensed', '=', False)]}</attribute>
```

Comment:
It think that you should override ``_compute_visible_expense_policy`` and let the attrs as it is defined on sale (https://github.com/odoo/odoo/blame/master/addons/sale/views/sale_views.xml#L1169).

Could you adapt ? 

Thanks.

Reply:
Sure I'll do it that way. I hesitated during dev since it seemed weird to me to have to rewrite attrs like this.
Glad that your review goes in this sense.

Reply:
:+1: =)